FinniversKit/Sources/Recycling/GridViews/Markets/GridView/Cell/MarketsGridViewCell.swift

```diff
@@ -8,7 +8,7 @@ class MarketsGridViewCell: UICollectionViewCell {
     // MARK: - Internal properties
 
     private let cornerRadius: CGFloat = 16
-
+    
```

Comment:
changes such as these where just a line happens to be added are best discarded since they don't add much value 🙂